lib/diagnostic.js

```diff
@@ -5,17 +5,27 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
-const min = function (/* previous, current */) {
-
+const min = function (originalArray, callback, initalValue) {
```

Comment:
This function is being passed into ```reduce()``` as a callback function.  Its parameters should be the accumulator (or ```previous``` element in this case), the ```current``` element, and the current working array (omitted here).  In this particular case, ```previous``` and ```current``` are assumed to be numbers.